frontend/src/_layouts/base.html

```diff
@@ -1,118 +1,161 @@
+      <a href="https://open.spotify.com/show/2YcWMvVkZXYMgfXOq907Bm?si=cOkhRfbISxe6_38OYMCA8w&nd=1" target="_blank">
+        <iconify-icon icon="ph:spotify-logo-bold" class="text-xl"></iconify-icon>
+      </a>
+      <!--Spotify Icon-->
+      <a href="https://podcasts.google.com/feed/aHR0cHM6Ly9hbmNob3IuZm0vcy8zY2Y4ZDRmMC9wb2RjYXN0L3Jzcw=="
+        target="_blank">
+        <iconify-icon icon="mdi:google-podcast" class="text-xl"></iconify-icon>
+      </a>
+      <!--Google Podocast Icon-->
+      <a href="https://purebhakti.us2.list-manage.com/subscribe?u=cff0da0dff6847d2469bdff2b&id=b2107d9953"
+        target="_blank">
+        <iconify-icon icon="quill:inbox-newsletter" class="text-xl"></iconify-icon>
+      </a>
+      <!--Subscribe Newsletter Icon-->
+      <!--Closed Icons.-->
+    </div>
+  </section>
+  <!--Top section Join us now-->
+  <!--Navigation Bar and Logo-->
+  <section class="w-full border-b px-4 bg-white border-b-white md:border-b-gray-200">
+    <nav class=" flex items-center xl:container lg:w-full justify-between flex-wrap py-4 mx-auto ">
+      <!--Logo-->
+      <div><img src="~/src/images/logo.png" alt="" class="w-2/3 md:w-full"> </div>
+      <!--Logo-->
+      <!--Menu-->
+          <nav
+            class="md:space-x-8 text-sm text-stone-900 font-semibold uppercase !mt-10 md:!mt-0 !space-y-3 md:!space-y-0">
+            {%- assign menu = collections.topnav | eleventyNavigation -%}
+            {%- for entry in menu -%}
+              <a
+                href="{{ entry.url }}"
+                {% if entry.url == page.url %}
+                class="border-b-4 h-8 md:inline block  justify-center border-b-white hover:text-green opacity-100 hover:border-b-4 hover:border-b-green active [&.active]:text-green active [&.active]:border-b-green"
+                {% endif %}
+                {% if entry.target == 'external' %}
+                  target="_blank"
+                {% endif %}
+              >
+                {{ entry.title }}
+              </a>
+            {%- endfor -%}
+        </nav>
         </div>
-      </div>
+      </aside>
+      <!--Menu-->
     </nav>
-    <!-- banner -->
-    {% if bannerShow != false %}
-      <div class="mx-auto flex w-full max-w-screen-xl flex-wrap items-center justify-center space-x-1 border-b px-4 py-2 text-sm sm:pl-36 md:pl-56 lg:pl-72">
-        <a class="inline-flex items-center rounded-md bg-red-400 px-2 py-1 text-white shadow-sm" href="/join">JOIN US</a>
-        <span class="py-1 text-center">for <strong>Daily Harikatha &amp; Short Videos</strong> in Hindi &amp; English </span>
-      </div>
-    {% endif %}
-    <div class="mx-auto w-full max-w-screen-xl flex-1">
-      <div class="mt-4 flex-none text-center font-header text-sm font-bold uppercase text-teal-800 sm:hidden">{{ title }}</div>
-      {{ content }}
+  </section>
+  <!--Navigation Bar and Logo-->
+
+   <!--Main body section-->
+  <div class="xl:container w-full mx-auto md:my-6 ">
+    <div class="md:flex">
```

Comment:
See if we can eliminate this nesting and merge the `class` attributes into one `div`.